packages/react-components/react-swatch-picker-preview/src/utils/renderUtils.tsx

```diff
@@ -0,0 +1,31 @@
+import * as React from 'react';
+import { ColorSwatch, ColorSwatchProps, SwatchPickerRow } from '../';
+
+export const renderSwatchPickerRow = (
+  colors: ColorSwatchProps[],
+  rowIdx: number,
+  SwatchElement: React.ElementType = ColorSwatch,
+) => {
+  return (
+    <SwatchPickerRow key={`row-${rowIdx}`}>
+      {colors.map((color, index) => (
+        <SwatchElement key={`${color.value}-${index}`} {...color} />
```

Comment:
This key is invalid:
- using `index` in keys is the antipattern
- `color.value` should be unique anyway, so it should be good enough

Reply:
Fixed. What about row component on line 10? It doesn't have unique value, would it be ok to use index there?

Reply:
Yeah, that's unfortunately our single option there